src/pages/CreatePersona/CreateCard/CreateCard.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC, useCallback, useEffect } from 'react';
+import { useLazyQuery, useMutation } from '@apollo/react-hooks';
+import { useHistory } from 'react-router-dom';
+
+import {
+  CHANGE_PERSONA_STEP_PATH,
+  GET_CARD,
+  GET_PERSONA_STEP_PATH,
+  GetCardType,
+  GetCurrentStepType,
+} from 'global/graphqls/Persona';
+import { TopNav } from 'components/TopNav/TopNav';
+
+import { WideButton } from '../components/WideButton';
+
+export const CreateCard: FC = () => {
+  const [getInitializedState, { data }] = useLazyQuery<GetCurrentStepType>(GET_PERSONA_STEP_PATH);
+  const [getCard, cardData] = useLazyQuery<GetCardType>(GET_CARD);
+  const [personaStepPath] = useMutation(CHANGE_PERSONA_STEP_PATH);
+  const history = useHistory();
+
+  const onNextClick = useCallback((): void => {
+    personaStepPath({ variables: { personaStepPath: 'page' } }).then(_r => {
+      history.push({
+        pathname: 'page',
+      });
+    });
+  }, [personaStepPath, history]);
+
+  useEffect(() => {
+    getInitializedState();
+    getCard();
+  }, [getInitializedState, getCard]);
+
+  useEffect(() => {
+    console.error(cardData.data);
```

Comment:
Do we need it?
console.error(cardData.data);

Reply:
this is a WIP work so this will be used to init form later

Reply:
this is reworked, please review it once more time